libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.scss

```diff
@@ -9,6 +9,13 @@
   display: flex;
 }
 
+/* Hide errors for the datepickers since they're now shown under the ID select. */
+.sky-date-range-datepicker-wrapper {
+  ::ng-deep sky-form-errors {
```

Comment:
Is there a way we can do this without `::ng-deep`, like through the inputs of the child component that displays form errors?

Reply:
I added a new, internal `errorsScreenReaderOnly` input to `SkyInputBoxComponent`.